src/sonic-yang-models/tests/yang_model_tests/tests/feature.json

```diff
@@ -14,5 +14,10 @@
     },
     "FEATURE_WITH_NO_OWNER" : {
         "desc": "Config feature table without set_owner"
+    },
+    "FEATURE_WITH_INVALID_BOOLEAN_TYPE" : {
```

Comment:
>FEATURE_WITH_INVALID_BOOLEAN_TYPE [](http://example.com/codeflow?start=5&length=33)

The order of sections is different from original PR. Suggest align with original, so future cherry-pick will be easier.

Reply:
Done